table/partitioned_filter_block.cc

```diff
@@ -13,27 +13,27 @@ namespace rocksdb {
 PartitionedFilterBlockBuilder::PartitionedFilterBlockBuilder(
     const SliceTransform* prefix_extractor, bool whole_key_filtering,
     FilterBitsBuilder* filter_bits_builder, int index_block_restart_interval,
-    PartitionedIndexBuilder* const part_index_builder)
+    PartitionedIndexBuilder* const p_index_builder)
```

Comment:
This is a constant pointer. Are you wanting to have a pointer to constant (e.g. const T*) ?